packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2235,11 +2236,17 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     /**
      * This is called to update objects whose routes are unused.
      * @param unusedRoutes - Data store and attachment blob routes that are unused in this Container.
+     * @returns - routes deleted from the runtime
      */
-    public updateUnusedRoutes(unusedRoutes: string[]) {
+    public updateUnusedRoutes(unusedRoutes: string[]): string[] {
```

Comment:
We should use a different function for sweep. First, the name updateUnusedRoutes does not reflect that the callee is supposed to delete nodes. Secondly, the semantics of sweep vs test mode are slightly different (for e.g., in test mode, a lot of validations done for sweep might not be needed).

Reply:
Would it be better to rename `updateUnusedRoutes`? That's all the function does

Reply:
How does `deleteObjects` sound?

Reply:
deleteUnusedRoutes or deleteUnusedNodes? since GC always communicates in those two terms.
I would create a different function for simplicity. Basically, don't touch the unused function and create a new one for sweep. We can then remove the unused one when we remove test mode.

Reply:
Ok, sounds good